pkg/kube/resources_test.go

```diff
@@ -44,7 +44,7 @@ func TestGetMultipleResourceFromSingleFile(t *testing.T) {
 
 	assert.Equal(t, 0, len(resources.Nodes), "Should not have any nodes")
 
-	assert.Equal(t, 4, len(resources.Controllers), "Should have four controllers")
+	assert.Equal(t, 1, len(resources.Controllers), "Should have four controllers")
```

Comment:
...why did this change? Also message is off now.

Reply:
Ahh I see the reason now. Thanks!

Reply:
It was actually wrong before apparently. 3 of those "Controllers" were empty yaml documents.